muikku-rest/src/main/java/fi/otavanopisto/muikku/rest/model/GuiderStudentRestModel.java

```diff
@@ -0,0 +1,28 @@
+package fi.otavanopisto.muikku.rest.model;
+
+import java.util.Date;
+import java.util.List;
+
+public class GuiderStudentRestModel extends Student {
+
+  public GuiderStudentRestModel() {
+  }
+
+  public GuiderStudentRestModel(String id, String firstName, String lastName, String nickName, String studyProgrammeName,
+      String studyProgrammeIdentifier, Boolean hasImage, String nationality, String language, String municipality,
+      String school, String email, Date studyStartDate, Date studyEndDate, Date studyTimeEnd, Date lastLogin,
+      String curriculumIdentifier, boolean updatedByStudent, Long userEntityId, List<StudentFlag> flags,
+      OrganizationRESTModel organization, boolean isUpperSecondarySchoolCurriculum) {
+    super();
```

Comment:
None of the base class variables given to this constructor are passed to the constructor of the base class, so they would be all null.